blazor/datagrid/sorting.md

```diff
@@ -264,23 +398,649 @@ In the following code example, custom SortComparer class was defined in the Cust
     }
 }
-```
+{% endhighlight %}
+{% highlight c# tabtitle="OrderData.cs" %}
+   public class OrderData
+    {
+        public static List<OrderData> Orders = new List<OrderData>();
 
-The following GIF represents custom SortComparer for CustomerID column. When the user clicks the CustomerID column's header, the custom SortComparer will sort the data according to the OrderID field value. So that custom SortComparer class sort the column data by using another column's value.
+        public OrderData()
+        {
 
-![Custom sort comparer in Blazor DataGrid](./images/blazor-datagrid-custom-sort-comparer.gif)
+        }
+        public OrderData(int? OrderID, string CustomerID, double? Freight, string ShipName)
+        {
+            this.OrderID = OrderID;
+            this.CustomerID = CustomerID;
+            this.Freight = Freight;
+            this.ShipName = ShipName;
+        }
 
-N> The SortComparer property will work only for local data.
-<br/> When using the column template to display data in a column, you will need to use the [Field](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_Field) property of [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html) to work with the [SortComparer](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_SortComparer) property.
+        public static List<OrderData> GetAllRecords()
+        {
+            if (Orders.Count() == 0)
+            {
+                int code = 10;
+                for (int i = 1; i < 2; i++)
+                {
+                    Orders.Add(new OrderData(10248, "VINET", 3.25, "Vins et alcools Chevali"));
+                    Orders.Add(new OrderData(10249, "TOMSP", 22.98, "Toms Spezialitäten"));
+                    Orders.Add(new OrderData(10250, "HANAR", 140.51, "Hanari Carnes"));
+                    Orders.Add(new OrderData(10251, "VICTE", 65.83, "Victuailles en stock"));
+                    Orders.Add(new OrderData(10252, "SUPRD", 58.17, "Suprêmes délices"));
+                    Orders.Add(new OrderData(10253, "HANAR", 81.91, "Hanari Carnes"));
+                    Orders.Add(new OrderData(10254, "CHOPS", 3.05, "Chop-suey Chinese"));
+                    Orders.Add(new OrderData(10255, "RICSU", 55.09, "Richter Supermarkt"));
+                    Orders.Add(new OrderData(10256, "WELLI", 48.29, "Wellington Import"));
+                    code += 5;
+                }
+            }
+            return Orders;
+        }
+
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public string ShipName { get; set; }
+        public double? Freight { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hDrUWMtqggCjRVvh?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
```

Comment:
having confusion in this example. need to reverify this

Reply:
Modified